index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
+                  <li>
+                    <div class="Css_3">
+                      <p>css</p>
+                    </div>
+                  </li>
+                </div>
+                <div class="textTagjava_3">
+                  <li>
+                    <div class="java_3">
+                      <p>javascript</p>
+                      </div>
+                  </li>
+                </div>
+              </ul>
+              <div id="Action_bttn_3">
+                <div class="projbttn_3">
+                  <div class="click_bttn_3">
+                  <button href="#SeeProject" >See Project</button>
+                </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="cardWork_4">
+            <img id="snap_portfolio_4"/>
+            <div class="leftBlock_4">
+              <div class="pryTxt_head_4">
+                <div class="text_head_4">
+                  Multi-Post
+                  Stories
+                </div>
+                <div class="frame_card_4">
+                  <div class="Client_card_4">
+                    <div class="Client_txt_4">
+                      CANOPY
+                    </div>
+                  </div>
+                  <span class="Counter_4"></span>
+                  <div class="Role_card_4">
+                    <div class="Role_txt_4">
+                      Back End Dev
+                    </div>
+                  </div>
+                  <span class="Counter2_4"></span>
+                  <div class="Year_card_4">
+                    <p class="Year_txt_4">
+                      2015
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="pryText_body_4">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+              </div>
+              <ul class="Tags_4">
+                <div class="textTag_Hml_4">
+                  <li >
+                    <div class="Hml_4">
+                      <p>html</p>
+                      </div>
+                  </li>
+                </div>
+                <div class="textTagcss_4">
+                  <li>
+                    <div class="Css_4">
+                      <p>css</p>
+                    </div>
+                  </li>
+                </div>
+                <div class="textTagjava_4">
+                  <li>
+                    <div class="java_4">
+                      <p>javascript</p>
+                      </div>
+                  </li>
+                </div>
+              </ul>
+              <div id="Action_bttn_4">
+                <div class="projbtn_4">
+                  <div class="click_bttn_4">
+                  <button href="#SeeProject" >See Project</a>
+                </div>
+                </div>
+              </div>
             </div>
           </div>
         </section>
```

Comment:
You currently do not have the About me section☹, kindly add it in line with the project requirements.☺